### PR TITLE
Drop upper version bounds on dependencies

### PR DESCRIPTION
**Pull request by JelleZijlstra**
They mostly cause unnecessary trouble.

### CODE REVIEW

**Review by ichard26 — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,10 @@
 - Unparenthesized tuples on annotated assignments (e.g
   `values: Tuple[int, ...] = 1, 2, 3`) now implies 3.8+ (#2708)
 
+### Packaging
+
+- All dependencies upper version bounds have been removed (#2718)
```
Comment (ichard26) [inline CHANGES.md:21]:
```suggestion
- All upper version bounds on dependencies have been removed (#2718)
```
I cannot do English grammar today 🙃

**Comment by ichard26** [comment]
I can't imagine Cooper would have a problem with this so I'm merging :)

**Comment by hartwork** [comment]
@JelleZijlstra thanks for that!

**Comment by dajose** [comment]
when is the next release including this change scheduled to? @ichard26

**Comment by ichard26** [comment]
By the end of this month currently.

**Pull request merged**